### ISSUE TITLE
Issue:Did you run npm install in Setup Wizard

### ISSUE THREAD

**Issue by IcedMango** [paperwork/paperwork#780]
Hi there,
  I have some problems in deploying paperwork.My web environment is using https://oneinstack.com 's. I've opened proc_open() and proc_get_status().My linux version is CentOS Linux release 7.3.1611 (Core) 
 I started installation from "Install Paperwork" to "Cleanup" on your wiki at https://github.com/twostairs/paperwork/wiki/Installation-----CentOS-Fedora .I've already tried npm install for many time but got nothing like that
![image](https://cloud.githubusercontent.com/assets/22059332/24260605/75b5eada-102f-11e7-9084-ed473020d64f.png)
Errors on Setup Wizard:
![image](https://cloud.githubusercontent.com/assets/22059332/24259808/f28eaf0e-102c-11e7-8b93-f4233dcd09bb.png)
![image](https://cloud.githubusercontent.com/assets/22059332/24259851/143c1b5a-102d-11e7-8936-47b004433c61.png)
when I use bower install it says that,so I added --allow-root
![image](https://cloud.githubusercontent.com/assets/22059332/24260232/3a06c9b0-102e-11e7-8806-ca0b518f0c1f.png)

Is there any thing else I can provide you to solve this problem?

**Comment by InCommingMessage** [comment]
I've already installed that dependencies.
![image](https://cloud.githubusercontent.com/assets/22059332/24281708/af0d2012-1095-11e7-9845-5a728247ef58.png)

**Comment by kuc** [comment]
You are installed them as `root`, so `www-data` or `nginx` user can't read them, I think.

**Comment by InCommingMessage** [comment]
@kuc Thanks for your suggestion , I'll try again later.

**Comment by Liongold** [comment]
@InCommingMessage Any updates?

**Comment by InCommingMessage** [comment]
@Liongold Nope. I've tried as kuc said but it still have errors like I mentioned.

**Comment by kuc** [comment]
@InCommingMessage `su` to `www-data` or `nginx` user, run again that `npm` command and check what errors are showing.

**Comment by sevmartinezjr1** [comment]
Per Selinux adding these booleans resolves the issues.

setsebool -P httpd_unified 1
setsebool -P httpd_can_network_connect 1
setsebool -P httpd_can_network_connect_db 1
setsebool -P httpd_execmem 1

Hope this helps.
Sev

**Comment by Liongold** [comment]
@InCommingMessage Still not fixed?

**Comment by InCommingMessage** [comment]
Sorry for not reply quickly.
@kuc I've tried su to www user, the dependencies shows the same result while I'm in root user.
@sevmartinezjr1 Thanks for your suggestion. I've already disabled SELinux after my system was ready.
![image](https://cloud.githubusercontent.com/assets/22059332/25619606/12b2da92-2f7e-11e7-853a-0d12b4e7a88d.png)
@Liongold Yep.Still not help. Maybe that's not compatible with my web environment ?
